src/helper.h

```diff
@@ -103,6 +103,11 @@ class helper
     /// \param needsNewServers true if new servers must started, false otherwise
     void handleKilledServer(HelperServerBase *srv, bool &needsNewServers);
 
+    /// Reacts to unexpected server death(s), including a failure to start server(s)
+    /// and an unexpected exit of a previously started server. \sa handleKilledServer()
+    /// \param madeProgress whether the died server responded to any requests
```

Comment:
Just to match the (correct) assumption (in the comment lines above) that we could be dealing with _multiple_ server deaths.

```suggestion
    /// \param madeProgress whether the died server(s) responded to any requests
```

Reply:
Done.